.github/workflows/update-conda-lockfile.yml

```diff
@@ -5,12 +5,6 @@ on:
   workflow_dispatch:
   schedule:
     - cron: "0 9 * * 1" # Mondays at 9AM UTC
```

Comment:
Should we make this run less frequently than weekly? I guess we should see how this cadence goes. We could do [1st/15th of each month](https://crontab.guru/#0_9_1,15_*_*) if weekly feels too frequent still.